app/models/child_benefit_tax_calculator.rb

```diff
@@ -46,7 +48,11 @@ def nothing_owed?
   end
 
   def has_errors?
-    errors.any? || starting_children.select { |c| c.errors.any? }.any?
+    errors.any? || starting_children_errors?
+  end
+
+  def starting_children_errors?
```

Comment:
@pmanrubia If `is_part_year_claim == 'no'` then the start and end dates for the child should default to the start and end dates for the tax year selected. User doesn't input any dates so there is no need for validation.